frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -214,7 +205,7 @@ const defaultConfig = {
     addIndex: [],
 } as ChartConfig;
 
-const emptyLayout = {} as Record<string, Record<string, unknown>>;
+const emptyLayout = {} as Partial<Layout>;
```

Comment:
Why need to changed it from `Record<string, Record<string, unknown>>` to `Partial<Layout>` though 🤔

Reply:
I think it's always better to have a more precise type

Reply:
👍and gives far more semantic!